app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -3,8 +3,40 @@ kind: registration
 name: Global Registration
 model: ProvisioningTemplate
 -%>
+<%
+    curl_opts = ["-H 'Content-Type: application/json'", "-H 'Accept: application/json'", "-H 'Authorization: Bearer #{@auth_token}'"]
+%>
+#!/bin/bash
+
+# Template parameters:
+<%= "#  - User: [#{@user.login}]" if @user %>
+<%= "#  - Organization: [#{@organization.name}]" if @organization %>
+<%= "#  - Location: [#{@location.name}]" if @location %>
+<%= "#  - Hostgroup: [#{@hostgroup.name}]" if @hostgroup %>
 
 if ! [ $(id -u) = 0 ]; then
     echo "Please run as root"
     exit 1
 fi
+
+OS=$(cat /etc/os-release | awk -F '=' '/^ID=/ {print $2}')
+HOST_NAME=$(hostname -f)
+
+echo "#"
+echo "# Running registration"
+echo "# Hostname: $HOST_NAME"
+echo "# OS: $OS"
+echo "#"
+
+if [ "$OS" == '"rhel"' ]; then
```

Comment:
If you source it and use bashisms anyway, why not go all the way and use the safe comparison? You also want to support CentOS. In the suggestion I chose to use ID_LIKE but I'm not sure if this is safe to assume. You also need to ensure subscription-manager is installed then.
```suggestion
if [[ $ID == rhel ]] || [[ $ID_LIKE == *rhel* ]] ; then
```